cmd/next.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+var (
+	errNotSupportedPart = errors.New("part string is not a supported next version increment")
+)
+
+var nextCmd = &cobra.Command{
+	Use:   "next part",
+	Short: "Next increment version based on semantic versioning",
+	Long: `Next increment version based on semantic versioning.
+Check latest version and increment part (major, minor, patch).
+Echo the next release version number to be used by CI tools or other commands like batch.`,
+	Args: cobra.ExactArgs(1),
+	RunE: runNext,
+}
+
+func init() {
```

Comment:
Not related to this line but could you add something to https://github.com/miniscruff/changie/blob/main/cmd/e2e_test.go, should be able to copy the `testLatest` method basically.

Reply:
Looks good